test/kitchen/test/integration/common/rspec/spec_helper.rb

```diff
@@ -205,10 +205,6 @@ def read_requirements(file_contents)
   reqs
 end
 
-def pip_freeze
```

Comment:
I don't know where we use this helpers, but are we sure we want to remove it instead of renaming it `pip2_freeze` ?

Reply:
It's not used anywhere

Reply:
OK